maquetado/setup/scss/tech.scss

```diff
@@ -8,21 +8,27 @@
   height: 300px;
   display: flex;
   flex-direction: column;  
-  align-items: center;
-
+  align-items: center;  
+  
   &::before {
     content: '';
     border: 2px solid  $green;
     position: absolute;
-    right: 5px;
-    bottom: 5px;
+    left: -7px;
+    top: -7px;
     z-index: -1;
     border-radius: 10px;
     width: 210px;  
     height: 300px;
+    transition: all 1s;  
```

Comment:
It will be better to select the attributes that are being modified for transition, in this case, left and top